web/template/create_document.gohtml

```diff
@@ -0,0 +1,207 @@
+            {{ end }}
+
+            {{ if .HasViewedInsertPage }}
+                <a href="create-document?id={{ .Case.ID }}&case={{ .Case.CaseType }}&templateId={{ .TemplateSelected.TemplateId }}" class="govuk-back-link">Back</a>
+
+                <h1 class="govuk-heading-l app-!-embedded-hide">Create Draft</h1>
+                <div class="govuk-form-group {{ if .Error.Field.selectOrGenerateRecipient }}govuk-form-group--error{{ end }}">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend">Select or generate a recipient</legend>
+                        {{ template "errors" .Error.Field.selectOrGenerateRecipient }}
+
+                        <form class="form" method="POST">
+                            <input type="hidden" name="id" value="{{ .Case.ID }}"/>
+                            <input type="hidden" name="case" value="{{ .Case.CaseType }}"/>
+                            <input type="hidden" name="templateId" value="{{ .TemplateSelected.TemplateId }}"/>
+                            <input type="hidden" name="hasViewedInserts" value="true"/>
+                            <input type="hidden" name="xsrfToken" value="{{ .XSRFToken }}"/>
+
+                            <div class="govuk-radios" data-module="govuk-radios">
+                                <div class="govuk-radios__item">
+                                    <input class="govuk-radios__input" id="f-recipientControls" name="recipientControls" type="radio" value="select" data-aria-controls="conditional-recipientControls">
+                                    <label class="govuk-label govuk-radios__label" for="f-recipientControls">
+                                        Select
+                                    </label>
+                                </div>
+                                <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-recipientControls">
+                                    {{ if eq (len .Recipients) 0 }}
+                                        <p class="govuk-body">No recipients available to select</p>
+                                    {{ else }}
+                                    <div class="govuk-!-width-one-half govuk-form-group {{ if .Error.Field.selectRecipient }}govuk-form-group--error{{ end }}">
+                                        <div class="govuk-form-group {{ if .Error.Field.selectRecipient }}govuk-form-group--error{{ end }}">
+                                            <label class="govuk-label" for="f-selectRecipient">
+                                                Select recipient
+                                            </label>
+                                            {{ template "errors" .Error.Field.selectRecipient }}
+                                            <select class="govuk-select {{ if .Error.Field.selectRecipient }}govuk-select--error{{ end }}" id="f-selectRecipient" name="selectRecipient">
```

Comment:
In the current design you can select multiple recipients, though I'm not sure if that's used particularly regularly